### PR TITLE
move all jupyter-related parts to the notebooks-service

### PR DESCRIPTION
**Pull request by rokroskar**
This is to be used together with https://github.com/SwissDataScienceCenter/renku-notebooks/pull/40. The `renku-notebooks` PR needs to be merged before the tests here will pass.

### CODE REVIEW

File: charts/chartpress.yaml [hunk 1]
```diff
@@ -24,21 +24,21 @@ charts:
...
+      #     - ../services/jupyterhub
+      # singleuser:
+      #   contextPath: ../services/singleuser
+      #   buildArgs:
+      #     JUPYTERHUB_VERSION: "0.9.2"
+      #     RENKU_VERSION: "0.2.0rc1"
+      #   dockerfilePath: ../services/singleuser/Dockerfile
+      #   valuesPath: jupyterhub.singleuser.image
+      #   paths:
+      #     - ../services/singleuser
```
Comment (leafty) [inline charts/chartpress.yaml]: TODO: removve

File: charts/gitlab/Chart.yaml [hunk 2]
```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: '1.0'
 description: A Helm chart for the Renku Gitlab server
 name: gitlab
-version: 0.2.0-rc3
+version: 0.0-unclean-6e8b9df
```
Comment (leafty) [inline charts/gitlab/Chart.yaml]: Chartpress caught you

File: charts/minikube-values.yaml [hunk 3]
```diff
@@ -47,5 +31,29 @@ notebooks:
...
+    auth:
+      state:
+        enabled: true
+        cryptoKey: 86282f03d9886a64a46ee38f946e9ee27a600df4559584eeb90d0bfbd5a3dc0e
+      type: gitlab
+      gitlab:
+        clientId: jupyterhub
+        clientSecret: dummy-secret
+        callbackUrl: '""' # Forces the default callback url
+
```
Comment (leafty) [inline charts/minikube-values.yaml]: Use the same section, unmodified if possible

File: charts/renku/Chart.yaml [hunk 4]
```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: '1.0'
 description: A Helm chart for the Renku platform
 name: renku
-version: 0.2.0-rc3
+version: 0.0-unclean-6e8b9df
```
Comment (leafty) [inline charts/renku/Chart.yaml]: Chartpress caught you

File: charts/renku/requirements.yaml [hunk 5]
```diff
@@ -16,9 +16,9 @@ dependencies:
   repository: "https://swissdatasciencecenter.github.io/helm-charts/"
   condition: gitlab.enabled
 
-- name: jupyterhub
-  version: "v0.7-51c7a58"
-  repository: "https://jupyterhub.github.io/helm-chart"
+# - name: jupyterhub
+#   version: "v0.7-51c7a58"
+#   repository: "https://jupyterhub.github.io/helm-chart"
```
Comment (leafty) [inline charts/renku/requirements.yaml]: TODO: remove

File: charts/renku/values.yaml [hunk 6]
```diff
@@ -545,104 +545,104 @@ ui:
...
+#       enabled: false
+#   cull:
+#     enabled: true
+#     timeout: 86400
+#     every: 60
 apispec:
   image:
     repository: renku/apispec
-    tag: '0.2.0-rc3'
+    tag: '0.0-unclean-6e8b9df'
```
Comment (leafty) [inline charts/renku/values.yaml]: Chartpress caught you

File: charts/renku/values.yaml [hunk 7]
```diff
@@ -545,104 +545,104 @@ ui:
...
 
 tests:
   image:
     repository: renku/tests
-    tag: '0.2.0-rc3'
+    tag: '0.0-unclean-6e8b9df'
```
Comment (leafty) [inline charts/renku/values.yaml]: Chartpress caught you

File: charts/renku/values.yaml [hunk 8]
```diff
@@ -545,104 +545,104 @@ ui:
...
+#   proxy:
+#     service:
+#       type: ClusterIP
+#       nodePorts: {}
+#     https:
+#       enabled: false
+#   cull:
+#     enabled: true
+#     timeout: 86400
+#     every: 60
```
Comment (leafty) [inline charts/renku/values.yaml]: TODO: remove

Also, maybe keep some reference values in `notebooks.jupyterhub` section

File: scripts/minikube_deploy.py [hunk 9]
```diff
@@ -83,6 +83,7 @@ def main():
         helm_deploy_cmd = [
             'helm', 'upgrade', release,
             renku_chart,
+            '--debug',
```
Comment (leafty) [inline scripts/minikube_deploy.py]: Welcome to the matrix

File: charts/renku/values.yaml [hunk 10]
```diff
@@ -545,104 +545,104 @@ ui:
...
-    defaultUrl: /lab
-  # FIXME: bug in prepuller makes helm hang in certain cases. Fixed in 0.7
-  # so this should be removed eventually.
-  # https://github.com/jupyterhub/zero-to-jupyterhub-k8s/issues/477
-  prePuller:
-    hook:
-      enabled: false
-  auth:
-    type: gitlab
-    gitlab:
```
Comment (rokroskar) [inline charts/renku/values.yaml]: This auth.gitlab section should be uncommented and put under the notebooks part. It should then work on staging (though if there are other references to jupyterhub in values there, they need to have notebooks prepended).

**Comment by rokroskar** [comment]
@leafty I think the comments have been addressed

**Pull request merged**